id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/entity/CredentialEntity.java

```diff
@@ -74,6 +74,9 @@ public class CredentialEntity {
 	/** The deleted date time. */
 	@Column(name = "del_dtimes")
 	private LocalDateTime deletedDateTime;
+	
+	@Column(name = "status_comment")
+	private String statusComment;

```

Comment:
As per the default db schema of 1.1.4.4 we might not have this field, this change will cause consistence issue.